src/main/java/io/jenkins/plugins/pipeline_cloudwatch_logs/CloudWatchSender.java

```diff
@@ -129,17 +129,13 @@ private Object readResolve() {
     }
 
     @Override
-    public synchronized final PrintStream getLogger() {
+    public synchronized final OutputStream getOutputStream() {
```

Comment:
Prompted by the `bom` update: https://github.com/jenkinsci/workflow-api-plugin/pull/294